internal/store/postgres/org_billing_repository.go

```diff
@@ -403,11 +403,12 @@ func processStringDataType(filter rql.Filter, query *goqu.SelectDataset) *goqu.S
 		query = query.Where(goqu.L(fmt.Sprintf("coalesce(%s, '') = ''", filter.Name)))
 	case OPERATOR_NOT_EMPTY:
 		query = query.Where(goqu.L(fmt.Sprintf("coalesce(%s, '') != ''", filter.Name)))
-	case OPERATOR_IN, OPERATOR_NOT_IN:
-		// process the values of in and notin operators as comma separated list
-		query = query.Where(goqu.Ex{
-			filter.Name: goqu.Op{filter.Operator: strings.Split(filter.Value.(string), ",")},
-		})
+	case OPERATOR_IN:
+		// process the values of in operator as comma separated list
+		query = query.Where(goqu.Cast(goqu.I(filter.Name), "TEXT").In(strings.Split(filter.Value.(string), ",")))
```

Comment:
Are we supposed to give ,  separated strings and not an array ? Isn't array more semantically correct for an in operator (since we can't pass any value with `,` if we take as string.)

Reply:
RQL doesn't support Arrays yet. For now, we are going with comma-separated strings until there is proper support for Arrays in RQL.

Reply:
okay